stdlib/3/subprocess.pyi

```diff
@@ -2,37 +2,42 @@
 
 # Based on http://docs.python.org/3.6/library/subprocess.html
 import sys
-from typing import Sequence, Any, AnyStr, Mapping, Callable, Tuple, IO, Optional, Union, List, Type
+from typing import Sequence, Any, AnyStr, Mapping, Callable, Tuple, IO, Optional, Union, List, Type, Text
 from types import TracebackType
 
+_FILE = Union[int, IO[Any]]
```

Comment:
I brought this _FILE thing over from the python 2 stubs, though I'm actually not sure it's correct.